src/platform/site-wide/tests/accessible-menus.e2e.spec.js

```diff
@@ -158,7 +158,8 @@ const runTest = E2eHelpers.createE2eTest(client => {
   // client.assert.isActiveElement(hcFirstItem);
 
   // -- Non-sub menu buttons -- //
-  const prescriptionsLink = 'a[href="/health-care/prescriptions"]';
+  const prescriptionsLink =
+    'a[href="/health-care/refill-track-prescriptions/"]';
 
   client.click(benefitsButton);
```

Comment:
I think this whole file can be removed now, because this was a test for the non-React mega menu.